oasislmf/utils/inputs.py

```diff
@@ -85,7 +86,15 @@ def confirm_action(self, question_str, no_confirm=False):
         except KeyboardInterrupt:
             self.logger.error('\nexiting.')
 
-    def get(self, name, default=None, required=False, is_path=False):
+    def get_env(self, name, dtype=None, default=None):
+        prefix = 'OASIS'
+        env_var = os.getenv(f'{prefix}_{name.upper()}', default=default)
```

Comment:
As you don't reuse prefix in this function, I would hardcode it in the string or make it a global variable if we want to indicate that all env variables we use need to be prefixed with OASIS